packages/legacy/core/App/utils/crypto.ts

```diff
@@ -1,8 +1,12 @@
 import argon2 from 'react-native-argon2'
 
 export const hashPIN = async (PIN: string, salt: string): Promise<string> => {
-  const result = await argon2(PIN, salt, {})
-  const { rawHash } = result
+  try {
+    const result = await argon2(PIN, salt, {})
+    const { rawHash } = result
 
-  return rawHash
+    return rawHash
+  } catch (error) {
+    throw new Error('Error generating hash for PIN' + String(error))
```

Comment:
Nit for error readability: 
```suggestion
    throw new Error('Error generating hash for PIN: ' + String(error))
```